vllm/worker/model_runner.py

```diff
@@ -1041,6 +1042,7 @@ def capture(
         kv_caches: List[torch.Tensor],
         attn_metadata: AttentionMetadata,
         memory_pool,
+        stream,
```

Comment:
nit: Let's annotate the type here?

Reply:
fixed in [35b3351](https://github.com/vllm-project/vllm/pull/4818/commits/35b335153868371d79fa449767a12f97b70aeae3)